docs/storage.md

```diff
@@ -46,7 +53,12 @@ The directory structure of a Prometheus server's data directory will look someth
 ```
 
 
-Note that a limitation of the local storage is that it is not clustered or replicated. Thus, it is not arbitrarily scalable or durable in the face of disk or node outages and should be treated as you would any other kind of single node database. Using RAID for disk availability, [snapshots](querying/api.md#snapshot) for backups, capacity planning, etc, is recommended for improved durability. With proper storage durability and planning, storing years of data in the local storage is possible.
+Note that a limitation of local storage is that it is not clustered or
+replicated. Thus, it is not arbitrarily scalable or durable in the face of
+drive or node outages and should be managed like any other single node
+database. RAID is suggested for storage availability and [snapshots](querying/api.md#snapshot)
+are recommended for backups, capacity planning, etc. With proper
```

Comment:
The capacity planning doesn't fit here, as written it makes it seem related to snapshots.

Reply:
Rephrasing to take the original "capacity planning" out.